tests/integration/tests/test_standalone_agent_integration.py

```diff
@@ -0,0 +1,105 @@
+"""
+This test is a basic integration test for cloudbeat.
+The test executed on pre-merge events as required test.
+The following flow is tested:
+Cloudbeat -> ElasticSearch
+"""
+import time
+import json
+import pytest
+import allure
+from commonlib.io_utils import FsClient
+
+testdata = ['file', 'process', 'k8s_object']
+CONFIG_TIMEOUT = 30
```

Comment:
increase timeout to 60 to compensate for agent startup

Reply:
fixed